src/modernizer/violations.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<!--
```

Comment:
Let's remove the license header (we don't do that for other XML files which are configuration) and format this with four spaces (like other XML files).